docs/references/cli.rst

```diff
@@ -3,23 +3,49 @@
 CLI
 ===
 
-PostgREST provides a CLI with the commands listed below:
+PostgREST provides a CLI with the options listed below:
+
+.. code:: text
+
+  Usage: postgrest [-v|--version] [-e|--example] [--dump-config | --dump-schema] 
+                 [FILENAME]
+
+    PostgREST 13.1 (pre-release) / create a REST API to an existing Postgres
+    database
+
+  Available options:
+    -h,--help                Show this help text
+    -v,--version             Show the version information
+    -e,--example             Show an example configuration file
+    --dump-config            Dump loaded configuration and exit
+    --dump-schema            Dump loaded schema as JSON and exit (for debugging,
+                             output structure is unstable)
+    FILENAME                 Path to configuration file
+
+Config
+------
+
+.. code:: bash
+
+  $ postgrest /path/to/postgrest.conf
+
+Runs PostgREST with the given :ref:`file_config`. See :ref:`configuration` for information on configurations.
```

Comment:
Otherwise this looks redundant with [Config File](https://postgrest--4349.org.readthedocs.build/en/4349/references/configuration.html#file-config). Also`FILENAME` has relation with the above output.

```suggestion
FILENAME
--------------

Runs PostgREST with the given :ref:`file_config`.
```